src/LogFileReader.cpp

```diff
@@ -74,12 +74,19 @@ bool LogFileReader::readAll(const QString& fileName)
 // File input is a csv file in the format of hh:mm:ss:zzz, voltage, current
 bool LogFileReader::parseLine(const QString& line, BatteryData& batteryData) const
 {
-    // TODO implement this first
-
-    // This is here to the compiler happy. Otherwise the compile
-    // will have an error warning about an unused variable. Remove this
-    // when you use it.
-    Q_UNUSED(line);
-    Q_UNUSED(batteryData);
+    QString::SectionFlag flag = QString::SectionSkipEmpty;
+    QString _bdt = line.section(",",0,0, flag);
+    QTime bdt = QTime::fromString(_bdt, "hh:mm:ss.zzz");
```

Comment:
In general writing strings in non print functions, as well as straight numbers in functions calls is poor structure. Normally, you would either derive such a value from somewhere else, or you would give it a descriptive name as a constant variable somewhere. 

In this case, the `"hh:mm:ss.zzz"` constant, since it can't be derived elsewhere, it should made as a constant. Normally how we do constants in C++ on the solar car team is adding a namespace at the top. So the code would be something like:
```c++
namespace 
{
    const QString SOME_DESCRIPTIVE_NAME = "hh:mm:ss.zzz";
}
```
I would suggest something similar for the line `QString _bdt = line.section(",",0,0, flag);`